spec/views/gsis/show.html.slim_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'spec_helper'
+require 'helpers/json_format_helper'
+
+describe 'gsis/show' do
+  let(:model_class) { User }
+  let(:variable_to_assign) { :gsi }
+  let(:expected_fields) { %w(id name email) }
+
+  it_behaves_like 'a JSON show view:'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.